src/transaction.js

```diff
@@ -772,20 +779,30 @@ Transaction.prototype.runStream = function(query) {
 
 /**
  * Determines whether or not this Transaction should be retried in the event
- * of an ABORTED error.
+ * of a retryable error.
  *
  * @param {error} err - The request error.
  * @return {boolean}
  */
 Transaction.prototype.shouldRetry_ = function(err) {
   return (
-    err.code === ABORTED &&
+    this.isRetryableErrorCode_(err.code) &&
     is.fn(this.runFn_) &&
     Date.now() - this.beginTime_ < this.timeout_ &&
     err.metadata.get(RETRY_INFO_KEY).length > 0
   );
 };
 
+/**
+ * Specifies whether a specific error code can be retried.
+ *
+ * @param {number} errCode - the error code
+ * @return {boolean}
+ */
+Transaction.prototype.isRetryableErrorCode_ = function(errCode) {
```

Comment:
Can you add a `describe` test block for `isRetryableErrorCode_`?

- it should return true for an aborted error code
- it should return true for an unknown error code